README.md

```diff
@@ -291,7 +291,7 @@ backend {
 }
 ```
 
-Or, via `-Dbackend.abortJobsOnTerminate=true` command line option.
+Or, via `-Dserver.abortJobsOnTerminate=true` command line option.
```

Comment:
Hmm okay.  Can you think of a better choice than `server` that encompasses both?